context/app/static/js/shared-styles/buttons/DisabledButton/DisabledButton.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+
+function DisabledButton({ disabled, ...props }) {
```

Comment:
- Worried that `DisabledButton` may be confusing, because if disabled is false, it's not actually "disabled"... Maybe just `Button`, and then the only thing that needs to change is the import?
- Can you link to the Safari/MUI issue here, or some discussion about this? Is the idea that when there's a fix upstream, we can just go back to using regular `Button`?
- Maybe file a followup issue, if that's the plan, link to that too, in the TODO? I like there to be a bidirectional connection between TODOs and issues.

Reply:
I'd prefer to use a name that distinguishes this instead of `Button`. Some places say 'disableable' is a word, but a bit of a jumble. Yes the idea is to go back to using regular `Button` once it is fixed. I sent you a link to the issue. I'll add a TODO and an issue.